collection-scripts/gather_mr

```diff
@@ -2,6 +2,10 @@
 # shellcheck disable=SC1091
 source common.sh
 resources=("modelregistries")
```

Comment:
```suggestion
resources=("modelregistries" "deployments" "services")
```
Add deployments and services created by model registry operator from modelregistry custom resource.

Reply:
These are default kinds, including out of the box, no need specify again